aws_longer/__init__.py

```diff
@@ -107,14 +107,18 @@ def handle_cleanup(arguments):
     except keyring.errors.PasswordDeleteError:
         pass
 
+def get_aws_username() -> str:
```

Comment:
Can you move this before `get_token` so that it's sorted? Also, can you run `black` on the project so that it sticks to style formatting? I'll add a test to ensure that is check for.

Reply:
done